helm/hpcc/templates/roxie.yaml

```diff
@@ -234,7 +234,7 @@ spec:
         roxie-cluster: {{ $roxie.name | quote }}
         accessDali: "yes"
         accessEsp: "yes"
-        helmVersion: 8.8.0-closedown0
+{{ include "hpcc.addStandardLabels" (dict "root" $ "component" "server" "name" $roxie.name "instance" $servername) | indent 8 }}
```

Comment:
component name "server" is too generic, elsewhere (in roxie.yaml) it is called "roxie-server"

Reply:
roxie-server for app roxie is a bit redundant, but that's fine we'll shoot for consistency